src/__tests__/style-component-test.js

```diff
@@ -105,4 +105,28 @@ describe('<Style> component', () => {
       }
     `);
   });
+
+  it('adds scopeSelector if no selectors are present', () => {
+    const output = TestUtils.renderIntoDocument(
+      <Style
+        rules={{
+          color: 'red',
```

Comment:
Add a test with two props, I'd expect them to be merged under the single scope selector.